### PR TITLE
feat:new theme

### PR DESCRIPTION
**Pull request by littlecxm**
I use [squidfunk/mkdocs-material](https://github.com/squidfunk/mkdocs-material) for new theme and adjust some element
since mkdocs's pdf export feature is so powerful
and i also sort python dependency into `requirements.txt` for generic project used
I also re-design a new [favicon](https://github.com/littlecxm/armbian-docs/blob/944825c7e5434613c4f7871b561066347f845187/docs/images/logo.png) for docs website
screenshot:
image
dark mode:
image

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -1,16 +1,46 @@
...
+        - navigation.tabs
+        - navigation.top
+        - toc.integrate
 
-theme:
-  name: readthedocs
-  highlightjs: true # enable syntax highlight for code blocks
+extra:
+  analytics:
+    provider: google
```
Comment (EvilOlaf) [inline mkdocs.yml:35]: What is this for?

Comment (littlecxm) [reply mkdocs.yml:35]: for theme intergrated google analyze code extension

Comment (igorpecovnik) [reply mkdocs.yml:35]: UA-284946-9 is for docs.armbian.com

Comment (EvilOlaf) [reply mkdocs.yml:35]: ok

**Comment by EvilOlaf** [comment]
@lanefu are backend changes necessary if we marge this?

**Comment by lanefu** [comment]
Ah let's bite the bullet and add the missing automation to upload to our docs site.

This is sweet.

**Comment by lanefu** [comment]
Can probably add chroot jail pretty easy

**Comment by lanefu** [comment]
nevermidn can jail method later... did the moviosu thing and ran installed requirements txt on server :)

**Comment by lanefu** [comment]
@littlecxm very awesome.. thank you...  deployed!

**Comment by igorpecovnik** [comment]
looks a loot better! Many thanks!

**Comment by littlecxm** [comment]
np :)

**Pull request merged**